lefthk/src/main.rs

```diff
@@ -70,23 +69,23 @@ fn main() {
 }
 
 fn send_command(command: &impl Command) {
-    let path = errors::exit_on_error!(BaseDirectories::with_prefix(lefthk_core::LEFTHK_DIR_NAME));
+    let path = BaseDirectories::with_prefix(lefthk_core::LEFTHK_DIR_NAME);
     let pipe_name = Pipe::pipe_name();
     let pipe_file = errors::exit_on_error!(path.place_runtime_file(pipe_name));
     let mut pipe = fs::OpenOptions::new().write(true).open(pipe_file).unwrap();
```

Comment:
We are catching if there is an issue with placing the pipe file, so this should be all good.